syntax = "proto3";

package todoPackage;


/**
A simple Todo service that lets the client add todo items and returns them when asked for.
*/
service Todo{
    rpc createTodo (TodoItem) returns (TodoItem);
    rpc readTodos (Empty) returns (TodoItems);
}

message Empty {}

message TodoItem{
    int32 id=1;
    string text=2;
}

message TodoItems{
    repeated TodoItem items=1;
}